README.md

```diff
@@ -1 +1,30 @@
-# to-do-rpoject
\ No newline at end of file
+# To-Do-List
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo
+
+
+### Prerequisites
+
+Modern web browser with Javascript enabled
+
+## Authors
+
+👤 **Alexandre Medina**
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Microverse for the opportunity
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- In order to make your repo look more professional, please include steps on how a beginner can run your project locally 👍 👍